analyses/oncoprint-landscape/run-oncoprint.sh

```diff
@@ -72,18 +84,30 @@ Rscript --vanilla 00-map-to-sample_id.R \
   --filename_lead ${primaryplus_filename} \
   --independent_specimens ../../data/independent-specimens.wgs.primary-plus.tsv
 
-Rscript --vanilla 01-plot-oncoprint.R \
-  --maf_file ${intermediate_directory}/${primaryplus_filename}_maf.tsv \
-  --cnv_file ${intermediate_directory}/${primaryplus_filename}_cnv.tsv \
-  --fusion_file ${intermediate_directory}/${primaryplus_filename}_fusions.tsv \
-  --metadata_file ${histologies_file} \
-  --png_name ${primaryplus_filename}_oncoprint.png
+# Print oncoprints by broad histology
+for histology in "Low-grade astrocytic tumor" \
+"Embryonal tumor" \
+"Diffuse astrocytic and oligodendroglial tumor" \
+"Ependymal tumor" \
+"Other CNS"
+do
 
-# Genes of interest only version of oncoprint
-Rscript --vanilla 01-plot-oncoprint.R \
-  --maf_file ${intermediate_directory}/${primaryplus_filename}_maf.tsv \
-  --cnv_file ${intermediate_directory}/${primaryplus_filename}_cnv.tsv \
-  --fusion_file ${intermediate_directory}/${primaryplus_filename}_fusions.tsv \
-  --metadata_file ${histologies_file} \
-  --goi_list ${genes_list} \
-  --png_name ${primaryplus_filename}_goi_oncoprint.png
+  Rscript --vanilla 01-plot-oncoprint.R \
+    --maf_file ${intermediate_directory}/${primaryplus_filename}_maf.tsv \
+    --cnv_file ${intermediate_directory}/${primaryplus_filename}_cnv.tsv \
+    --fusion_file ${intermediate_directory}/${primaryplus_filename}_fusions.tsv \
+    --metadata_file ${histologies_file} \
+    --png_name ${primaryplus_filename}_"$histology"_oncoprint.png \
+    --broad_histology "$histology"
+
+  # Genes of interest only version of oncoprint
+  Rscript --vanilla 01-plot-oncoprint.R \
+    --maf_file ${intermediate_directory}/${primaryplus_filename}_maf.tsv \
+    --cnv_file ${intermediate_directory}/${primaryplus_filename}_cnv.tsv \
+    --fusion_file ${intermediate_directory}/${primaryplus_filename}_fusions.tsv \
+    --metadata_file ${histologies_file} \
+    --goi_list ${genes_list} \
+    --png_name ${primaryplus_filename}_"$histology"_goi_oncoprint.png \
+    --broad_histology "$histology"
```

Comment:
Can we put this in the same loop as the primary only steps? I can't think of a reason not to at this stage.

Reply:
Good point, done!